css/custom.css

```diff
@@ -54,3 +54,30 @@
     padding: 7px;
     box-sizing: content-box;
 }
+
+/* Media Queries for YouTube Video */
+/* ytVideo is ID for video */
+@media (min-width: 992px) {
+    #ytVideo {
+        width: 880px;
```

Comment:
Don't use media queries and static values, use bootstraps responsive classes like s12

Reply:
Ohh okay, sorry I'll fix it